src/routes/routes.tsx

```diff
@@ -1,25 +1,35 @@
 import { createBrowserRouter, NavigateOptions } from 'react-router-dom';
 
-import { CompareAccounts, Homepage, Signup } from '@/pages';
+import { CompareAccounts, Homepage, Redirect, Signup } from '@/pages';
 
-type TRoutes = '/' | '/signup' | '/compare-accounts';
+export const routes = {
+    home: '/',
+    signup: '/signup',
```

Comment:
This url should be absolute?
https://deriv.com/signup

Reply:
@habib-deriv it works for TH_v3, I just moved these strings to the object. Also, I don't understand why did you write  about deriv.com, this PR is for trader's hub repo